xbmc/peripherals/bus/virtual/PeripheralBusAddon.cpp

```diff
@@ -330,14 +355,6 @@ void CPeripheralBusAddon::GetDirectory(const std::string &strPath, CFileItemList
     addon->GetDirectory(strPath, items);
 }
 
-void CPeripheralBusAddon::OnEvent(const ADDON::AddonEvent& event)
-{
-  if (typeid(event) == typeid(ADDON::AddonEvents::Enabled) ||
-      typeid(event) == typeid(ADDON::AddonEvents::Disabled) ||
-      typeid(event) == typeid(ADDON::AddonEvents::InstalledChanged))
-    UpdateAddons();
```

Comment:
I know should be done now with new "ADDON::AddonEvents::ReInstalled".

But again, informated about reinstall after the old becomes deleted and still on memory in use.